services/src/main/java/org/fao/geonet/api/records/formatters/Register.java

```diff
@@ -145,6 +145,7 @@ public JSONObject serviceSpecificExec(HttpServletRequest request,
             return response;
         } finally {
             IO.deleteFile(uploadedFile, false, Geonet.FORMATTER);
+            context.clear();
```

Comment:
I see some code in the API package creating a `ServiceContext`, but not calling the `clear` method at the end.

Is this due to the way of creating the `ServiceContext` (`serviceManager.createServiceContext` vs `ApiUtils.createServiceContext`)?

Reply:
I expect more instances of clear() can be added over time, especially if you can confirm how it is used.